common/metrics/interfaces.go

```diff
@@ -33,6 +33,13 @@ import (
 )
 
 type (
+	// Stopwatch is a helper for simpler tracking of elapsed time, use the
+	// Stop() method to report time elapsed since its created back to the
+	// timer or histogram.
```

Comment:
```suggestion
	// Stopwatch is an interface for tracking of elapsed time, use the
	// Stop() method to report time elapsed since its created back to the
	// timer or histogram.
```